FirebasePerformance/Sources/FPRClient.m

```diff
@@ -79,17 +87,18 @@ + (void)load {
       FPRLogError(kFPRClientInitialize, @"Failed to initialize the client with error:  %@.", error);
     }
 
-    [notificationCenter removeObserver:listener];
-    listener = nil;
+    *isCacheable = YES;
+
+    return [self sharedInstance];
   };
 
-  // Register the Perf library for Firebase Core tracking.
-  [FIRApp registerLibrary:@"fire-perf"  // From go/firebase-sdk-platform-info
-              withVersion:[NSString stringWithUTF8String:kFPRSDKVersion]];
-  listener = [notificationCenter addObserverForName:kFIRAppReadyToConfigureSDKNotification
```

Comment:
Not familiar with this but purely based on diff, with the `kFIRAppReadyToConfigureSDKNotification` removed, I'm guessing this is the same behavior replaced by `instantiationTiming:FIRInstantiationTimingEagerInDefaultApp` when registering components, is that right?